### DISCUSSION TITLE
Is aldente supposed to always show up in the dock?

### DISCUSSION THREAD

**Discussion by Delvien** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#706] [Q&A] [answered]
I don't really want it to, and I cant hide it, or close it out without shutting down aldente.

**Comment by MatthiasKerbl** [comment, marked as answer]
Hi @Delvien,

You can turn this feature off. Just go to AlDente/Settings/Appearance/Show Dock Icon -> Disable.

Cheers,
Matthias

**Reply by Delvien** [reply]
Son of a gun, i combed through 10 times and I did not see that. Thank you, fixed the issue.

**Comment by francescostella** [comment]
@MatthiasKerbl 

Is 'Show Dock Icon' removed?

I have AlDente Pro 1.28 on macOS 15.3.1 (24D70), trough Setapp. 

Has been quite a while AlDente is showing in my Dock, I have been waiting to some updates to fix it but it didn't, hence the reason to reach out here.
![CleanShot 2025-02-19 at 19 04 58](https://github.com/user-attachments/assets/4acb1ad6-38c7-4eaf-a52d-2ee193afec66)

**Comment by francescostella** [comment]
@MatthiasKerbl 
Sorry my fault. Just after I posted here, I realized through the repo tags v1.28 was not the latest, but there is actually a v1.31 for Setapp.
The weird thing is that Setapp hasn't been update automatically AlDente automatically, even if it is keeping updated other apps.
So I removed AlDente and re-installed, and this fixed the 'Show Dock Icon' toggle, now is showing up.

I'm wrote this post for posterity. Here the screenshot:
![CleanShot 2025-02-19 at 19 19 29](https://github.com/user-attachments/assets/4e4cf08c-3732-45f8-be47-9f8885971072)
